clients/fides-js/src/fides-tcf.ts

```diff
@@ -104,7 +100,8 @@ const updateExperience = ({
 /**
  * Initialize the global Fides object with the given configuration values
  */
-const init = async (config: FidesConfig) => {
+async function init(this: FidesGlobal, config: FidesConfig) {
```

Comment:
Hah, wow - TIL about Typescript `this` parameters: https://www.typescriptlang.org/docs/handbook/2/functions.html#declaring-this-in-a-function

Interesting